src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/EnergyDistribution/LevelScattering/Builder/test/Builder.test.cpp

```diff
@@ -0,0 +1,52 @@
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
+#include "ACEtk.hpp"
+
+using namespace njoy::ACEtk;
+using namespace njoy::ACEtk::interpretation;
+
+SCENARIO( "Testing EnergyDistribtion::LevelScattering::Builder" ){
+    ContinuousEnergyNeutron::Builder CENBuilder;
+
+    auto parentBuilder = CENBuilder.reaction( 14 ).energyDistribution();
+    using LSBuilder = decltype( parentBuilder.levelScattering() );
+
+    struct TestBuilder : public LSBuilder {
+      using LSBuilder::construct;
+      using LSBuilder::LSBuilder;
+    };
+
+  TestBuilder lSBuilder( parentBuilder );
+
+  GIVEN( "valid inputs" ){
+    double awr{ 1.1 };
+    double QValue{ 2.2 };
+    lSBuilder.atomicWeightRatio( awr )
+             .QValue( QValue );
+
+    auto distribution = lSBuilder.construct();
+
+    THEN( "the values of the distribution can be checked" ){
+      CHECK( awr == distribution.atomicWeightRatio );
+      CHECK( QValue == distribution.QValue );
+
+      AND_THEN( "the contents can be ACE-ified" ){
+        std::vector< double > aceified{};
+        aceified.push_back( QValue );
+        aceified.push_back( awr );
```

Comment:
Depends on what Q and A are, if I follow the formulas: this will be incorrect (see above).

Reply:
Fixed.